### PR TITLE
Makes the clownroid into a vault

### PR DESCRIPTION
**Pull request by MadmanMartian**
See title. Makes the clownroid into a vault so it's not as easy to cheese endgame materials such as phazon and bananium using 3 minutes of telescience

Because the derelict mommi will not be happy, their material synth now makes silver, gold, uranium, diamond, and plasma.

todo:
 - [ ] There has to be a better way of removing that defunct mommi material synth

:cl:
 * tweak: Makes the clown asteroid into a vault
* tweak: Soviet MoMMIs now have a fully scanned material synth

### CODE REVIEW

**Comment by ghost** [comment]
Make phazon unreplicatable and keep roid as is.

**Comment by brndd** [comment]
Why is there phazon in the clown roid anyway? I mean it's fine if it's a vault, but if it were to stay a non-vault I'd at least be for removing the phazon from it leaving it only with bananium (and uranium and whatever else spawns there).

**Comment by boy2mantwicethefam** [comment]
already tried tho

**Comment by ghost** [comment]
I agree. lol 30 mins Phazon mech gets tiring after the tenth time or so. You should have to ACTUALLY explore to find Phazon, not do the tired leap from Derelict Chapel down

**Comment by SonixApache** [comment]
so this is version 2 of #22224
in that one I asked for your motivation and in this one you gave it and I totally agree so :+1: from me

**Comment by Kitfox88** [comment]
remove phazon REMOVE PHAZON

**Comment by Caboozles** [comment]
And so ends an era of fun.

**Comment by Chazach** [comment]
phazon as wacky effects juice: fun and not really powergay since half the time you become allergic to oxygen or move so fast you cant enter doors, much less click people
phazon as a tool to make murder mechs: shit
solution: remove the shitty murder mechs

**Comment by Caboozles** [comment]
Most murder-bone mechs are durands. Phazons are rarely used in comparison.

File: code/modules/mob/living/silicon/mommi/mommi_modules.dm [hunk 1]
```diff
@@ -72,7 +72,11 @@
 /obj/item/weapon/robot_module/mommi/soviet/New(var/mob/living/silicon/robot/R) //Powercreep!
 	..()
 
-
+	var/atom/A = locate(/obj/item/device/material_synth/robot/mommi) in src
```
Comment (jknpj) [inline code/modules/mob/living/silicon/mommi/mommi_modules.dm]: AAAAAAAAAAAAAAAAAAAAAAAAAAA

I was supposed to fix this kind of spaghetti by now but you could just remove the default synth for the default mommi module and add it to the NT child. It's not a "fix" but for now it'll work.

**Comment by SonixApache** [comment]
@Caboozles 
durands are extremely weak and shitty, even *with* the energy reflection nonsense
gygaxes are actually optimized for mech-on-mook combat simply because they're fast

however phazons are extremely powerful, painfully common and a hard counter to basically any antag (and for antags, a hard counter to basically anything a crew can throw at them)

**Comment by Caboozles** [comment]
You can call them shitty, but in the past week the same guy with a durand with the same loadout killed:

-The entire sec team (4 members)
-The captain
-A borg with a laser
-Various crew with ions and lasers

And in a different round
-10 xenos at the same time on a shuttle

The problem is more murdermechas take almost no power drain.

**Comment by Lunstra** [comment]
Tbh while I do support this PR because fuck phazon mechs, chazach has a point, maybe this is just a bandaid over how busted phazon mechs are, the other uses of phazon are just fun stuff like making floor tiles and becoming allergic to oxygen and having 10 max health. I'd say HONKs are bullshit too but they did catch a nerf on their ear rape spam iirc and honestly who in their right mind even makes one of those motherfuckers for the clown anyway?

**Comment by ghost** [comment]
I'd definitely agree Mechs need to be nerfed, but let's not get ahead of ourselves. Gating off the powergamey Phazon mechs from being asspulled reliably every round is the first step

**Comment by brndd** [comment]
Both the phazon and clown mechs are supposed to be special rare items and are balanced around being special rare items. The materials for making them being hardmapped in is idiotic and does nothing but invite powergay, as has been evidenced by Phazons being in so many rounds as of late.

**Comment by KingofHONK** [comment]
downvoted because this is a lazy way of fixing the core issue of mechas not having an accessible counter.

**Comment by Caboozles** [comment]
I seriously contend the reason Phazons are considered so bad is the same reason mechs need to be nerfed. There is absolutely no concern about power, which is supposed to be the big drain on the Phazon itself.

But when you have 50,000 cell power and can fit 4 modules on your mech, cell power becomes not an issue and Mechs go from being heavy hitters to suits of armor that have no counter because the pilot essentially has gained 200 power.
I love Phazons because they're fast and fun and look great, but the reason they're so hard to deal with isn't because they've got health, it's that there's no cost for their mobility.

Let's seriously look for a second.
-Phazing is 200 energy/tick.
-Repair module is 100 energy/tick
-Energy Relay is (I think?) 100 energy/tick

And in combat
-Laser Cannon is 60 energy for 60 damage.
-Scattershot is 25 energy for 20 shots of 10 damage.
-Energy Relay is again, possibly 100 energy/tick.

You functionally can keep the repair module going at all times if you have an energy relay to balance it out. Even if you don't, it'll take you nearly 500 ticks to go from 50,000 to 0 with the repair module constant.
Phazons simply have too many slots, don't have any power drain, and a few seconds out of combat can repair them to normal with no interaction needed from the pilot. There is never a reason to turn the repair relay off unless you're really trying to optimize your energy flow.

I seriously think the solution here is to dramatically increase energy costs across the board for combat-based mecha actions.  Meaning they can remain to be those heavy hitters, but can't be used 24/7 like they currently are.
Make Phazing be 500/tick or so. Energy Relay down to 50/tick. Repair up to 250/tick.
Laser cannons to 150/shot, Scattershots to 50 or something. Make recharge stations charge slower. Something, anything to make energy a real consideration for what you're attempting to do than something you glance at every 3 minutes because you left the phaze array on and accidentally walked across the entire station, woops!

I seriously think the 50,000 cell should also be harder to get. It literally has no use other than for mechs, which has led to this problem to begin with.

**Comment by brndd** [comment]
I think the dynamic of running out of cell charge and then having to walk all the way back to a recharger and sit there for five minutes is stupid and bad gameplay. It'd be much nicer if mechs instead had some sort of a capacitor system (think mana pool, or the capacitor from Eve if you're familiar with that) that drains fast but also replenishes on its own reasonably fast.

But these are concerns for the mech rework. Even if you fixed all the balance issues with the mechs, leaving the clown roid in as it is would mean that the two mechs specifically designed to be rare, almost easter egg-like things are basically available at round start. It's antithetical to their design.

**Comment by Durgen** [comment]
\>tweak

this is balance

**Comment by Anton-Kr** [comment]
tfw gimmick of stealing spider shuttle and giving robotics bannanium + phazon 30 mins into the round is broken. 
Time to make it into a bar or something I guess.

**Comment by DrSnips** [comment]
Speaking of. Either make the shuttle go to the clown roid always or give it a new GOOD location to dock. I don't want sonething lazy like the IOU vault. And speaking of the IOU vault remove it when?

**Comment by jknpj** [comment]
>And speaking of the IOU vault remove it when?

IOU a vault removal

But yeah, if there's a disk leading to the clown roid, it needs to lead you to the vault, if it exists.

**Comment by brndd** [comment]
if you remove the shit vaults what are you left with? sokoban and that's about it.

**Comment by DrSnips** [comment]
There are shit vaults and then there's the IOU vault below that. Its a really shit vault.

**Comment by Kitfox88** [comment]
remove mech REMOVE MECH

**Comment by Kammerjunk** [comment]
vaults that do nothing are a-okay

File: code/modules/mob/living/silicon/mommi/mommi_modules.dm (hunk 1 above)
Comment (MadmanMartian) [reply code/modules/mob/living/silicon/mommi/mommi_modules.dm]: Fix the spaghet!

**Comment by DrSnips** [comment]
There are plenty of flavor vaults that have no real loot and Im fine with that, like the polar vault, but the IOU vault is just utter shit. It's shape and color is ugly, it's basicly empty save a few mimics, and it has next to no flavor besides LOL MAGIC GANG GANG paper.

**Comment by spoonylove** [comment]
Ill remember fondly sprinkling some phazon on bread and handing it out

**Comment by Anton-Kr** [comment]
for real tho could we add something to buff the salvage shuttle into mattering after this change. Maybe have it start with some of the vault disks I dunno

**Comment by brndd** [comment]
IIRC the salvage shuttle can't even dock at most of the vaults.

**Comment by Blithering** [comment]
Destinations disks are neat, its just a shame that the only way to obtain them is to have already been to the location they take you to, and then there's seldom any reason to return unless its spessmart

**Comment by aceedex** [comment]
Merriment forbidden

**Pull request merged**